Sources/Private/Experimental/ExperimentalFeatureConfiguration.swift

```diff
@@ -16,4 +16,8 @@ public struct ExperimentalFeatureConfiguration {
   /// animate without executing on the main thread every frame.
   public var useNewRenderingEngine = false
```

Comment:
nit: remove default value since it's specified in the initializer so there's a single source of truth
```suggestion
  public var useNewRenderingEngine: Bool
```

Reply:
Good idea